packages/umi-uploader-irys/package.json

```diff
@@ -28,9 +28,9 @@
     "test": "ava"
   },
   "dependencies": {
-    "@irys/sdk": "^0.0.2",
+    "@irys/sdk": "0.2.10",
```

Comment:
Any reason for removing the `^` prefix which makes the version range follow semver?

Reply:
Nope, my extension just removes them apparently.

Reply:
Gotcha. Same issue for `bignumber.js` btw.

Reply:
Thanks! Gonna have to see if I can turn that setting off 😅